### PR TITLE
Wei du

### PR DESCRIPTION
**Pull request by chef27615**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -22,8 +22,12 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <div>
+          <Route exact path='/' component ={MovieList} />
```
Comment (jheinz1999) [inline client/src/App.js:26]: Awesome work adding in Routes!

File: client/src/Movies/Movie.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ export default class Movie extends Component {
 
   componentDidMount() {
     // change this line to grab the id passed on the URL
-    const id = 1;
+    const {id} = this.props.match.params;
```
Comment (jheinz1999) [inline client/src/Movies/Movie.js:14]: Great job using destructuring here!

File: client/src/Movies/MovieList.js [hunk 3]
```diff
@@ -34,7 +36,7 @@ export default class MovieList extends Component {
 function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
   return (
-    <div className="movie-card">
+    <Link to={`/movies/${movie.id}`} className="movie-card">
```
Comment (jheinz1999) [inline client/src/Movies/MovieList.js:39]: Awesome work getting the Link to work! Note that you can also use `history.push()` to link to another page without wrapping everything in a Link component

**Review by jheinz1999 — APPROVED** [review]
Looking good Wei! I definitely challenge you to take a look at some of the stretch problems!

File: client/src/Movies/MovieList.js (hunk 3 above)
Comment (chef27615) [reply client/src/Movies/MovieList.js:39]: thank you for the advise, I will give it a try, and I am attempting stretch now

**Pull request merged**